manager/integration/tests/test_recurring_job.py

```diff
@@ -1239,100 +1224,199 @@ def test_recurring_job_delete(client, batch_v1_beta_api, volume_name):  # NOQA
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back2)
     wait_for_cron_job_delete(batch_v1_beta_api, JOB_LABEL+"="+back3)
 
-    # Should be able to delete recurring job while existing in volume label
+
+@pytest.mark.recurring_job  # NOQA
+def test_recurring_job_delete_should_remove_volume_label(client, batch_v1_beta_api, volume_name):  # NOQA
+    """
+    Scenario: test delete recurring job should remove volume labels
+
+    Given 1 volume created.
+    And  create `snapshot1` recurring job.
+         create `backup1`   recurring job.
+    And  `snapshot1` job exist in volume recurring job label.
+         `backup1`   job exist in volume recurring job label.
+
+    When delete `snapshot1` recurring job.
+    Then `snapshot1` job not exist in volume recurring job label.
+         `backup1`   job     exist in volume recurring job label.
+
+    When delete `backup1` recurring job.
+    Then `snapshot1` job not exist in volume recurring job label.
+         `backup1`   job not exist in volume recurring job label.
+
+    Given create `snapshot1` recurring job with `group-1` in groups.
+    And   create `snapshot2` recurring job with `group-1` in groups.
+    And   create `backup1`   recurring job with `group-1` in groups.
+    And   remove volume recurring job label `default` job-group.
```

Comment:
Group `default` will be applied to the volume automatically, but Longhorn won't do this for group `group-1`. Need to clarify this part here.

Reply:
Added comment for clarification.

Reply:
You haven't mentioned that `group-1` is added to the volume

Reply:
Fixed.